src/Database/LSMTree/Internal/Config.hs

```diff
@@ -271,7 +271,7 @@ data DiskCachePolicy =
        --
        -- Use this policy if the expected access pattern for the table
        -- has good temporal locality for recently inserted keys.
-     | DiskCacheLevelsAtOrBelow !Int
+     | DiskCacheLevelZeroTo !Int
```

Comment:
The comment still says "at or below"

Reply:
Fair enough. I didn't bother because I have already completely rewritten that comment on my own branch.